api/v1beta1/disruption_cron_webhook.go

```diff
@@ -121,6 +121,11 @@ func (d *DisruptionCron) ValidateCreate() (_ admission.Warnings, err error) {
 		return nil, err
 	}
 
+	// send validation metric
```

Comment:
could we not have these comments? they don't explain

Reply:
yeah to be fair I've copy pasted what we had on the disruption validation. I'll remove them